lib/charms/layer/docker_registry.py

```diff
@@ -431,10 +441,11 @@ def start_registry(name=None, run_args=None):
                         level=hookenv.ERROR)
             raise
     else:
-        # NB: config determines the port, but the container always listens to 5000
+        # NB: config determines the port, but the container always listens to 5000 and
+        # 5001
         # https://docs.docker.com/registry/deploying/#customize-the-published-port
         cmd = ['docker', 'run', '-d', '-p', '{}:5000'.format(port),
-               '--restart', 'unless-stopped']
+               '-p', '{}:5001'.format(debug_port), '--restart', 'unless-stopped']
```

Comment:
Ahh, okay so we're port-forwarding the debug-port into the hard-coded port `5001` from above